buildSrc/src/main/java/org/elasticsearch/hadoop/gradle/buildtools/info/BuildParams.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
```

Comment:
We may need to double check if these files are ok to be listed with the Elastic license or if they need to be converted to Apache for this project. It's likely that this license is fine since it does not get published from the project as an artifact, but it should be double checked for compliance.

Reply:
We can relicense this as Apache 2.0 for simplicity. We are the copyright owners so we are free to do so at our discretion.

Reply:
Updated the licenses to apache 2 as the rest of the sources in this repo.